scripts/should-build.rb

```diff
@@ -0,0 +1,108 @@
+require 'json'
+
+PLATFORM = ENV.fetch('task', 'unknown')
+
+def sh(cmd)
+  puts "$ #{cmd}"
+  output = `#{cmd}`.chomp
+  puts output
+  output
+end
+
+def git_branch
+  sh('git rev-parse --abbrev-ref HEAD')
+end
+
+# diff two hashes at a path and return the changed items
+def deps_diff(old_hash, new_hash, path)
+  (new_hash.dig(*path).to_a - old_hash.dig(*path).to_a).to_h
+end
+
+NPM_DEP_NAME_REGEXP = /react|jsc/.freeze
```

Comment:
Can we get some whitespace between this and the method that follows?

Reply:
are you sure we can't just inline this? I really want to inline this again… so concise…

Reply:
Yes, I'm sure.  We print it out elsewhere.

Reply:
although then we can't inline it into the other string… bah.